templates/base.html

```diff
@@ -75,6 +75,7 @@
               </a>
               <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
                 <a class="dropdown-item" href="{{ config.base_url | safe }}/about/rse/">RSE</a>
+                <a class="dropdown-item" href="{{ config.base_url | safe }}/about/membership/">Membership</a>
```

Comment:
maybe call it "How to join"?

Reply:
It also tells you what membership means and what the benefits are. Maybe the "become a sponsor" link could be replaced by "how to join" and point to /about/membership?

Reply:
ok I think "membership" is a good name then, yes I agree the become a sponsor in the purple box should be replaced

Reply:
Replaced it on the front page